src/Components/TruckCard/TruckCard.js

```diff
@@ -5,6 +5,7 @@ import notDownvoted from '../../assets/notDownvoted.png'
 import notUpvoted from '../../assets/notUpvoted.png'
 import downvoted from '../../assets/downvoted.png'
 import upvoted from '../../assets/upvoted.png'
+import { NavLink } from 'react-router-dom';
```

Comment:
I'm working in this Component at the moment so there's going to be some merge conflicts here, but I'll get it worked out once my CSS PR is up